mgwr/sel_bw.py

```diff
@@ -200,8 +200,9 @@ def __init__(self, coords, y, X_loc, X_glob=None, family=Gaussian(),
         self.search_params = {}
 
     def search(self, search_method='golden_section', criterion='AICc',
-            bw_min=None, bw_max=None, interval=0.0, tol=1.0e-6, max_iter=200, init_multi=True,
-            tol_multi=1.0e-5, rss_score=False, max_iter_multi=200):
+            bw_min=None, bw_max=None, interval=0.0, tol=1.0e-6, max_iter=200,
```

Comment:
In what situation would a user want to have different max iters for multi and single?

Reply:
Not sure, I've never had to tinker with either, but they do control different loops. max_iters is for each individual golden section and max_iters_multi is for the gam backfitting. Probably safe to set them to the same value, but if you did want to change one I am not sure you would want to change both.